projects/kit/src/components/callout/components/callout-close-button/callout-close-button.component.html

```diff
@@ -0,0 +1 @@
+<pupa-button-icon (click)="closeCallout()" [icon]="'app-cross'" [kind]="'secondary'" [size]="'xs'"></pupa-button-icon>
```

Comment:
nit: нет смысла чтобы ангуляр вычислял строки:
```suggestion
<pupa-button-icon (click)="closeCallout()" icon="app-cross" kind="secondary" size="xs"></pupa-button-icon>
```

Reply:
Исправил